.circleci/config.yml

```diff
@@ -329,33 +329,15 @@ jobs:
             - v2-playwright
       - show-large-files-and-directories
       - run: "cd integration-tests && npm install"
-      - run: scripts/build/_generate-etags
-      - run:
-          name: Prep test split
-          command: |
-            # get full list of tests
-            integration-tests/test-list.sh > rundir/all-tests
-
-            # split them using timing info
-            TESTS=$(circleci tests split --split-by=timings --timings-type=testname rundir/all-tests)
-
-            # concat them into a pattern (note: $TESTS is deliberately unquoted)
-            PATTERN=$(printf -- "%s|" $TESTS)
-
-            # remove last char
-            PATTERN=${PATTERN%?}
-
-            echo $PATTERN > test-pattern
+      - regenerate-etags
       - run:
           name: Run integration tests
           command: |
             # Run the server first to set up the DB correctly for the prep script
             scripts/run-fsharp-server --published
             # Wait for users to be added so prep.sh works
             ./scripts/devcontainer/_wait-until-apiserver-ready
-            integration-tests/run.sh --concurrency=3 --retry --pattern="`cat test-pattern`" --published
-            rm test-pattern
-
+            integration-tests/run.sh --concurrency=6 --retry --published
```

Comment:
I believe this will have no effect. Integration tests are running [in serial](https://github.com/darklang/dark/blob/02f2783f1104fd9ad751b5f8dee982b671bd012c/integration-tests/tests.ts#L100) currently

Reply:
Thank you, that confused me.

Reply:
I'll test it - I thought it was running at 3x and we might get 6x, but I'm less optimistic of any speedup now.

Reply:
Neat that it worked for you! (removing the .serial)
For me, it kept yielding flaky tests, but looks good for you :)

Reply:
So it actually didn't work - I can't get more than 1 worker (at least on my local machine).

Reply:
> Running 70 tests using 1 worker

Same in CI

Reply:
> 70 passed (1m)

Just had this locally with --concurrency=16

Reply:
? hilarious.

Reply:
Really curious why I was seeing _significant_ flakiness before, seemingly due to ApiServer strain, and you're not now.
But, happy just seeing the result.

Reply:
Maybe you weren't using the `--retry` flag, which is enabled in CI?

Reply:
it was failing for me in CI specifically.

Reply:
Curious. If it starts flaking in CI again, we can rollback.